mloop/utilities.py

```diff
@@ -55,7 +55,9 @@ def config_logger(**kwargs):
 
 def _config_logger(log_filename = default_log_filename,
                   file_log_level=logging.DEBUG,
+                  file_log_string='%(asctime)s %(name)-20s %(levelname)-8s %(message)s',
                   console_log_level=logging.INFO,
+                  console_log_string='%(levelname)-8s %(message)s',
```

Comment:
Let's add these new arguments to this function's docstring, and let's add them to `logging_config.txt` in the examples directory as well

Reply:
Ha!  I didn't even know about the `.txt` example files since they are not used in the `Labscript` interface.   In any case the docstring and `logging_config.txt` have both been updated.

Reply:
Yep! It's used in [the official M-LOOP docs](https://m-loop.readthedocs.io/en/v3.3.4/examples.html#logging), which are generated from the repo itself - primarily the stuff in the `docs` directory. So we should generally try to keep the info there updated in tandem with the changes in your upcoming PRs